### PR TITLE
Adding new article

### PR DESCRIPTION
**Pull request by andrewBrunette**
Adding article for SMS Passcode

### CODE REVIEW

File: Billing/billing-process-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{{{
```
Comment (chrislittle) [inline Billing/billing-process-guidelines.md]: why are you trying to add this kb that was rejected previously again?

File: Ecosystem Partners/Marketplace Guides/getting-started-with-savant-wms-blueprint.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+﻿{{{
```
Comment (chrislittle) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-savant-wms-blueprint.md]: did you mean to add savant too?

File: Ecosystem Partners/Marketplace Guides/getting-started-with-smspasscode-template.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  - Additional Notes or work to be done: ########
+
+  Please let me know if you have any questions or issues. Kindly send me a reply once the work has been completed and let us know the IP address of the server where this technology has been deployed.
+
+  Thank you very much,
+
+	   Your_Name_Here
+	   ```
+Your account alias and PIN are available from your account info page and your user profile page respectively.
+
```
Comment (chrislittle) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-smspasscode-template.md]: your code text is not ending properly.  please review the preview as it requires fixing

File: Ecosystem Partners/Marketplace Guides/getting-started-with-smspasscode-template.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  - Server Name to import image as: ##########
+  - VLAN in the account to add the Server to: ########
+  - Additional Notes or work to be done: ########
+
+  Please let me know if you have any questions or issues. Kindly send me a reply once the work has been completed and let us know the IP address of the server where this technology has been deployed.
+
+  Thank you very much,
+
+	   Your_Name_Here
+	   ```
```
Comment (jim-phillips) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-smspasscode-template.md]: @andrewBrunette  Fix alignment of \`\`\` This is putting the rest of the file as a code block.

**Pull request merged**